sktime/classification/base.py

```diff
@@ -62,9 +62,7 @@ def fit(self, X, y):
 
         Parameters
         ----------
-        X : 2D np.array (univariate, equal length series) of shape = [n_instances,
-            series_length]
-            or 3D np.array (any number of dimensions, equal length series) of shape =
+        3D np.array (any number of dimensions, equal length series) of shape =
```

Comment:
X: np.ndarray (3d array of shape (n_instances,n_dimensions,series_length)) or pd.DataFrame with each column a dimension, each cell a pd.Series (any number of dimensions, equal or unequal length series)